### PR TITLE
Remove linux wheels in favor of only manylinux wheels

### PR DESCRIPTION
**Pull request by brettlangdon**

### CODE REVIEW

File: scripts/build-dist [hunk 1]
```diff
@@ -22,6 +22,9 @@ done
 for whl in /dd-trace-py/dist/*-linux_${ARCH}.whl;
 do
   auditwheel repair "${whl}" -w /dd-trace-py/dist
+
+  # Remove linux wheel since we only want the manylinux wheels
+  rm "${whl}"
```
Comment (majorgreys) [inline scripts/build-dist]: Why not remove them from being built? 

https://github.com/DataDog/dd-trace-py/blob/master/scripts/build-dist#L15-L19

Comment (brettlangdon) [reply scripts/build-dist]: we need to build them first to then convert them into a manylinux wheel

Comment (brettlangdon) [reply scripts/build-dist]: build linux wheel -> convert to manylinux wheel

Comment (majorgreys) [reply scripts/build-dist]: 👍

**Pull request merged**